lib/gc/Transforms/GPU/LinalgToXeGPU.cpp

```diff
@@ -732,17 +732,17 @@ loadNdDescTiles(PatternRewriter &rewriter, Location loc, ValueRange loadTiles,
 
   VectorType vecLoadType =
       VectorType::get(tileType.getShape(), tileType.getElementType());
-  IntegerAttr vnniAxisAttr = nullptr;
+  UnitAttr vnniAxisAttr = nullptr;
   if (vnniConf) {
-    vnniAxisAttr = IntegerAttr::get(rewriter.getI64Type(), vnniConf->vnniAxis);
+    vnniAxisAttr = UnitAttr::get(rewriter.getContext());
     vecLoadType = getVnniVector(tileType.getShape(), tileType.getElementType(),
                                 *vnniConf);
   }
 
   SmallVector<Value> loadVec;
   for (auto tile : loadTiles) {
     auto loadOp = rewriter.create<xegpu::LoadNdOp>(
-        loc, vecLoadType, tile, vnniAxisAttr, transpose,
+        loc, vecLoadType, tile, vnniAxisAttr, transpose, nullptr,
```

Comment:
@dchigarev Sorry I am late for the party. The IMEX's XeGPU dialect interfaces are a bit different from the upstream one. If we compile GC+IMEX+Patched LLVM, the compiler complains here. Here in the PR, I have updated the code, just to make the compiler happy, but not yet pass the UTs related to this part of code.

Reply:
@Menooker thanks, I'm good with your changes.

I'm aware that the current linalg-to-xegpu pass is incompatible with the patched XeGPU dialect from IMEX. I'm currently working on a separate PR to them compatible.

Reply:
Good to know that you are aware of the issue. :)

Reply:
I just realized that this breaks the normal build for GPU. I think we should ifdef it or smth.

Reply:
I think we shouldn't include this pass in CPU builds at all (if `-DGC_USE_GPU=1`)

Reply:
> I think we shouldn't include this pass in CPU builds at all (if `-DGC_USE_GPU=1`)

I agree with you. Actually these changes in my PR will break the CPU CI. I will skip build the pass in cmake for CPU-only mode.